libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1201,7 +1201,7 @@ string YulUtilFunctions::storageArrayIndexAccessFunction(ArrayType const& _type)
 						let itemsPerSlot := div(0x20, <storageBytes>)
```

Comment:
This can be calculated at compile time, maybe we can swap it with template parameter and not calculate div here...